lib/utils/documentdb.ts

```diff
@@ -224,7 +224,11 @@ export function queryDocuments<T>(
   collectionUri: IDocumentDbCollectionUri,
   query: DocumentDb.DocumentQuery
 ): IResultIterator<T & DocumentDb.RetrievedDocument> {
-  const documentIterator = client.queryDocuments(collectionUri.uri, query);
+  const documentIterator = client.queryDocuments(collectionUri.uri, query, {
+    // enableCrossPartitionQuery is mandatory if you want to query for fields
+    // other than the partitionKey and Id
+    enableCrossPartitionQuery: true
```

Comment:
this has certainly performance implications, I would expose it as a parameter of `queryDocuments`, with a default value of `false`, letting the caller decide

Reply:
ok, I'll revert this back setting subscriptionId = serviceId